mongodb/bucketarchives.go

```diff
@@ -8,8 +8,9 @@ import (
 )
 
 type BucketArchive struct {
-	BucketKey string   `bson:"_id"`
-	Archives  Archives `bson:"archives"`
+	BucketKey            string         `bson:"_id"`
+	Archives             Archives       `bson:"archives"`
+	DefaultArchiveConfig *ArchiveConfig `bson:"default_archive_config"`
```

Comment:
This will be `nil` until the user sets it. We take advantage of that fact to detect `nil` and use the hard coded default instead.